src/utils.js

```diff
@@ -5,11 +5,26 @@ import {readFile as readFileCB} from 'fs';
 var readFile = Promise.promisify(readFileCB);
 
 /* Helper functions */
-const basename = filepath => path.basename(filepath, path.extname(filepath));
-const dirname  = filepath => path.normalize(path.dirname(filepath));
-const parentDirname = filepath => dirname(filepath).split(path.sep).pop();
-const removeNumbers = str  => str.replace(/^[0-9|\.\-]+/, '');
-const getFiles = glob => globby(glob, {nodir: true });
+
+function basename (filepath) {
+  return path.basename(filepath, path.extname(filepath));
+}
+function dirname (filepath) {
+  return path.normalize(path.dirname(filepath));
+}
+function parentDirname (filepath) {
+  return dirname(filepath).split(path.sep).pop();
+}
+function removeLeadingNumbers (str) {
+  return str.replace(/^[0-9|\.\-]+/, '');
```

Comment:
Are you comfortable with a possible breaking change? I think this pattern is better, too, but will it cause unexpected side effects or should we be all "full speed ahead, damn the torpedoes"?!